### PR TITLE
Implemented support for HOOMD v3+

### PR DESCRIPTION
**Pull request by mtortora**

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    - python==3.8
+    - numpy>=1.9
+    - scipy>=0.16
+    - matplotlib>=2.0
+    - pandas>=0.19
+    - hoomd>=3.0
+    - fresnel>=0.13
+    - gsd>=2.8
+    - pip:
+        - tornado==6.0.0
```
Comment (gfudenberg) [inline environment.yml]: what's this one for?

File: examples/tutorial.ipynb
Comment (gfudenberg) [inline examples/tutorial.ipynb:1]: maybe `chromosome_sizes` --> `chromosome_ends`?

also, would this more clear as a triple of (start, stop, topology) as in polychrom? or would some other format be more useful?

cc @golobor @rusch-emma

Comment (gfudenberg) [reply examples/tutorial.ipynb:1]: also thinking ahead to a format that would support some free-floating particles as well-- for a chain of length 1000 and 100 particles could this just be [1000,1001,1002, ... 1100], or would that be cumbersome down the line?

File: examples/tutorial_extrude.ipynb
Comment (gfudenberg) [inline examples/tutorial_extrude.ipynb:1]: I guess `steps_3D_to_1D` can move into the extrusion_dict.json?

Comment (gfudenberg) [reply examples/tutorial_extrude.ipynb:1]: nice visualization of loops w/ fresnel!

File: environment.yml (hunk 1 above)
Comment (mtortora) [reply environment.yml]: that's a fresnel dependency, presumably required to enable interactive viewing windows in jupyter (although I haven't yet managed to get those to work).
I had to downgrade it a bit, as versions 6.1+ appear to have broken things and lead to obscure server errors whenever calling fresnel

File: examples/tutorial.ipynb
Comment (mtortora) [reply examples/tutorial.ipynb:1]: The current format should already be flexible enough to support diffusible particles as well!
Here's an [extra example](https://github.com/open2c/polychrom-hoomd/blob/initial_commit/examples/tutorial_binders.ipynb) featuring free-floating binders, without any changes to the base code

**Pull request merged**